api.planx.uk/modules/webhooks/controller.ts

```diff
@@ -16,7 +16,7 @@ import { SanitiseApplicationData } from "./service/sanitiseApplicationData/types
 import { sanitiseApplicationData } from "./service/sanitiseApplicationData";
 
 export const sendSlackNotificationController: SendSlackNotification = async (
-  req,
+  _req,
```

Comment:
_Question:_

Am I right in saying `_req` is still being passed in despite not being used directly as down the chain it is required?

Reply:
That's correct, `_req` is always passed in, but no longer used in this instance. We're using the underscore prefix to indicate that this is an unused variable - there's a linting rule set up for this in `.eslintrc` under `@typescript-eslint/no-unused-vars`